test/contracts/promises/child-simple/wasm/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
```

Comment:
Having an extra /wasm crate for these low-level smart contracts is a bit overkill, but it's ok for now.
(The reason we have this in the framework is mostly for the coverage tools and other tools that don't know what to do with wasm).

Reply:
These contracts only partially import the framework and are somewhat low-level. If I remember correctly, the only way they could work was with an extra `wasm` crate. We can review this later and simplify these contracts.